### PR TITLE
chore(scripts): add helper scripts for testbench

### PR DESCRIPTION
**Pull request by Zelldon**
In order to not always try to find out what needs to be send to 

 * start a test bench run
 * to complete an analysis

I wrote some helper scripts, they can be extended of course with other stuff.
They expect that zbctl is installed in the PATH and a credentials file which should contain:

```
#!/bin/bash

set -euox pipefail

export ZEEBE_ADDRESS='*.zeebe.ultrawombat.com:443'
export ZEEBE_CLIENT_ID='***'
export ZEEBE_CLIENT_SECRET='***'
export ZEEBE_AUTHORIZATION_SERVER_URL='https://login.cloud.ultrawombat.com/oauth/token'
```
Can be copied from the client credentials in camunda cloud. I dont want to put this into git.

### CODE REVIEW

File: scripts/runTestbench.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (pihme) [inline scripts/runTestbench.sh:1]: please rename file to "scheduleDailyTestRuns` or something similar.

File: scripts/runTestbench.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+set -euox pipefail
+
+
+source credentials
+
+id=${1:-"zell"}
+generation=${2:-"Zeebe 0.24.4"}
```
Comment (pihme) [inline scripts/runTestbench.sh:9]: I think you can leave generation empty. Not 100% sure. There is one piece of case which takes the default geenration from the channel if no generation is given.

File: scripts/runTestbench.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -euox pipefail
+
+
+source credentials
+
+id=${1:-"zell"}
```
Comment (pihme) [inline scripts/runTestbench.sh:8]: I think this would be better without a default

**Review by pihme — APPROVED** [review]
Like the idea. Some nice to haves:
- add scripts to shellcheck test
- error messages when scripts are called without arguments

File: Jenkinsfile [hunk 4]
```diff
@@ -90,10 +90,16 @@ pipeline {
               dir('core/chaos-workers') {
                   println("Check all chaos script`s with shellcheck (linter).")
                   sh 'shellcheck -x *.sh'
+                  sh 'shellcheck -x *.sh'
```
Comment (pihme) [inline Jenkinsfile:93]:
```suggestion
                  sh 'shellcheck -x *.sh'
```

**Review by pihme — APPROVED** [review]

File: Jenkinsfile (hunk 4 above)
Comment (pihme) [reply Jenkinsfile:93]: Doppelt hält besser :-)

**Comment by Zelldon** [comment]
@pihme 
```
[2020-11-13T15:39:38.242Z] + shellcheck -x analysis-completed.sh scheduleDailyTestRun.sh

[2020-11-13T15:39:38.242Z] 

[2020-11-13T15:39:38.242Z] In analysis-completed.sh line 12:

[2020-11-13T15:39:38.242Z] source credentials

[2020-11-13T15:39:38.242Z]        ^---------^ SC1091: Not following: credentials: openBinaryFile: does not exist (No such file or directory)

[2020-11-13T15:39:38.243Z] 

[2020-11-13T15:39:38.243Z] 

[2020-11-13T15:39:38.243Z] In scheduleDailyTestRun.sh line 18:

[2020-11-13T15:39:38.243Z] source credentials

[2020-11-13T15:39:38.243Z]        ^---------^ SC1091: Not following: credentials: openBinaryFile: does not exist (No such file or directory)

[2020-11-13T15:39:38.243Z] 

[2020-11-13T15:39:38.243Z] For more information:

[2020-11-13T15:39:38.243Z]   https://www.shellcheck.net/wiki/SC1091 -- Not following: credentials: openB...
```

Cant check with shellcheck :laughing:

**Comment by Zelldon** [comment]
Any Idea/suggestions?

**Comment by Zelldon** [comment]
Found a solution :)

**Pull request merged**